src/components/inputs/SearchBar.tsx

```diff
@@ -35,7 +33,7 @@ export const SearchBar = ({ placeholder, setState, maxLength = 20 }: SearchBarTy
         onChange={(event) => handleChange(event)}
       />
       <div className={cx('searchbar-search-icon')}>
-        <Image src={url} alt={alt} width={16} height={16} />
+        <img className={cx('img')} src={url} alt={alt} />
```

Comment:
nesting 문법 사용한 classname 어떨까요 ㅎㅎ
```suggestion
        <img className={cx('searchbar-search-icon-img')} src={url} alt={alt} />
```

Reply:
넵 적용하겠습니다!